spec/exporters/conference_preferences_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+RSpec.describe Exporters::ConferencePreferences do
+
+  describe '#current' do
+    let!(:old_team) { create :team, :with_preferences, name: "OLDTEAM" }
+    let!(:new_team) { create :team, :with_preferences, :in_current_season, name: "NEWTEAM" }
+
+    subject { described_class.current }
+
+    it 'exports conference preferences for teams of the current season' do
```

Comment:
Cool! #823 is the dedicated issue. It is not super pressing, but since tests are equally part of the app as "normal" application code, they deserve the same amount of thoughts and love 🦄 

Feel free to ping me if anything is unclear about the issue.